cognite/transformations_cli/commands/utils.py

```diff
@@ -154,9 +154,13 @@ def print_jobs(jobs: List[TransformationJob]) -> str:
 T = TypeVar("T")
 
 
+def chunk_items(items: List[T], n: int = 10) -> Iterator[List[T]]:
```

Comment:
Unless 10 is important here, it seems we could simplify a lot by setting the default to 5.
Although, can't we just use batches of 10 everywhere?